### ISSUE TITLE
Reading states in intermediate d3plot files in case of adaptive meshing

### ISSUE THREAD

**Issue by ThinkZi** [qd-cae/qd-cae-python#40]
I am reading the d3plot files attached below using the D3plot class and except for the first and last file it seems that the results are not loaded to the D3plot.

This works fine and gives the expected result:

```
a= D3plot("//1_stage1_forming.d3plotao")
a.read_states(['strain', 'disp'])
t =a.get_partByID(1)
t.plot()
```
Note that "1_stage1_forming.d3plotao" is the last file in the tree, however if you read any of the files in between like "1_stage1_forming.d3plotad" or "1_stage1_forming.d3plotaa" and plot it, only the initial stage will be shown.
Also if you look at the elements of the intermediate d3plots and use a method like get_strain, it will return empty arrays, if you were to read one of the intermediate files.

```
t_elements = t.get_elements()
strains=[]
for elem in t_elements:
    strains.append(elem.get_strain())
strains[0]
```
array([], shape=(0, 0), dtype=float32)

Here are the files:

[1_stage1_forming.zip](https://github.com/qd-cae/qd-cae-python/files/2315304/1_stage1_forming.zip)
[1_stage1_forming_d3plot01.zip](https://github.com/qd-cae/qd-cae-python/files/2315317/1_stage1_forming_d3plot01.zip)
[1_stage1_forming_d3plotaa.zip](https://github.com/qd-cae/qd-cae-python/files/2315326/1_stage1_forming_d3plotaa.zip)
[1_stage1_forming_d3plotab.zip](https://github.com/qd-cae/qd-cae-python/files/2315337/1_stage1_forming_d3plotab.zip)
[1_stage1_forming_d3plotac.zip](https://github.com/qd-cae/qd-cae-python/files/2315338/1_stage1_forming_d3plotac.zip)
[1_stage1_forming_d3plotad.zip](https://github.com/qd-cae/qd-cae-python/files/2315339/1_stage1_forming_d3plotad.zip)
[1_stage1_forming_d3plotae.zip](https://github.com/qd-cae/qd-cae-python/files/2315340/1_stage1_forming_d3plotae.zip)
[1_stage1_forming_d3plotaf.zip](https://github.com/qd-cae/qd-cae-python/files/2315341/1_stage1_forming_d3plotaf.zip)
[1_stage1_forming_d3plotag.zip](https://github.com/qd-cae/qd-cae-python/files/2315342/1_stage1_forming_d3plotag.zip)
[1_stage1_forming_d3plotah.zip](https://github.com/qd-cae/qd-cae-python/files/2315343/1_stage1_forming_d3plotah.zip)
[1_stage1_forming_d3plotai.zip](https://github.com/qd-cae/qd-cae-python/files/2315352/1_stage1_forming_d3plotai.zip)
[1_stage1_forming_d3plotaj.zip](https://github.com/qd-cae/qd-cae-python/files/2315353/1_stage1_forming_d3plotaj.zip)
[1_stage1_forming_d3plotak.zip](https://github.com/qd-cae/qd-cae-python/files/2315354/1_stage1_forming_d3plotak.zip)
[1_stage1_forming_d3plotal.zip](https://github.com/qd-cae/qd-cae-python/files/2315355/1_stage1_forming_d3plotal.zip)
[1_stage1_forming_d3plotam.zip](https://github.com/qd-cae/qd-cae-python/files/2315356/1_stage1_forming_d3plotam.zip)
[1_stage1_forming_d3plotan.zip](https://github.com/qd-cae/qd-cae-python/files/2315357/1_stage1_forming_d3plotan.zip)
[1_stage1_forming_d3plotao.zip](https://github.com/qd-cae/qd-cae-python/files/2315361/1_stage1_forming_d3plotao.zip)
[1_stage1_forming_d3plotao01.zip](https://github.com/qd-cae/qd-cae-python/files/2315362/1_stage1_forming_d3plotao01.zip)

**Comment by qd-codie** [comment]
Will be officially fixed in 0.8.3. 

You can find a pre-release as always in the [dev branch dist folder](https://github.com/qd-cae/qd-cae-python/tree/dev/dist). Let me know it it also worked for you ❤️ 

![image](https://user-images.githubusercontent.com/5020407/44550154-afcde380-a723-11e8-81af-b9886184274a.png)

**Comment by ThinkZi** [comment]
Seems to be working, thanks
